### DISCUSSION TITLE
How can i make "in" queries against relationship field?

### DISCUSSION THREAD

**Discussion by Sipioteo** [pocketbase/pocketbase#2223] [Q&A] [answered]
The question is in the title, i tried, it is not working.
How can i do that?

In the pic, groups.members is the relation, obv

![Screenshot 2023-04-05 at 18 27 20](https://user-images.githubusercontent.com/39625454/230144295-1ab53c06-5f88-4e61-a968-87db5eb2a226.png)

**Comment by ganigeorgiev** [comment, marked as answer]
I assume that `groups.members` is a multiple `relation` field. PocketBase stores array values as serialized json array, so in this case you can use the SQLite [`json_each`](https://www.sqlite.org/json1.html#jeach) function to join each id element of the array and compare it against the `users.id`, something like:

```sql
SELECT ...
FROM groups
INNER JOIN json_each(groups.members) as je_members
INNER JOIN users on users.id = je_members.value
```
